pkgs/tools/misc/ttyrec/default.nix

```diff
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
 
   patches = [ ./clang-fixes.patch ];
 
-  makeFlags = [ "CFLAGS=-DSVR4" ]
+  makeFlags = stdenv.lib.optional buildPlatform.isLinux "CFLAGS=-DSVR4"
```

Comment:
Sorry I mixed this up, guys. I'll reread the docs before the next time I try to use it.